common/lib/xmodule/xmodule/html_module.py

```diff
@@ -417,6 +418,35 @@ class CourseInfoModule(CourseInfoFields, HtmlModuleMixin):
     # statuses
     STATUS_VISIBLE = 'visible'
     STATUS_DELETED = 'deleted'
+    TEMPLATE_DIR = 'courseware'
+
+    @XBlock.supports("multi_device")
+    def student_view(self, _context):
+        """
+        Return a fragment that contains the html for the student view
+        """
+        return Fragment(self.get_html())
+
+    def get_html(self):
```

Comment:
There should still be a docstring here though.  Please add an appropriate docstring in addition to the comment.